src/main/java/org/folio/bulkops/processor/ItemDataProcessor.java

```diff
@@ -111,6 +117,13 @@ public Updater<Item> updater(UpdateOptionType option, Action action) {
     };
   }
 
+  private void setupHoldingsData(Item item) {
+    item.setHoldingsData(String.join(HOLDINGS_LOCATION_CALL_NUMBER_DELIMITER,
+      isNull(item.getEffectiveLocation()) ? EMPTY : item.getEffectiveLocation().getName(),
+      isNull(item.getEffectiveCallNumberComponents()) ? EMPTY : item.getEffectiveCallNumberComponents().getCallNumber())
```

Comment:
This approach may be used when item's call number and locations are empty. Otherwise, you should retrieve them from holdings record.

Reply:
done